src/main/java/org/benetech/servicenet/adapter/linkforcare/LinkForCareDataMapper.java

```diff
@@ -0,0 +1,305 @@
+            StringUtils.equalsIgnoreCase(data.getServiceEligibilityMustMeetAge(), TRUE) ? "Must meet age guidelines" : null,
+            StringUtils.equalsIgnoreCase(data.getServiceEligibilityMustMeetDisability(), TRUE) ? "Must meet disability guidelines" : null,
+            StringUtils.equalsIgnoreCase(data.getServiceEligibilityMustMeetIncome(), TRUE) ? "Must meet income guidelines" : null,
+            StringUtils.equalsIgnoreCase(data.getServiceEligibilityMustBeVeteran(), TRUE) ? "Must be a veteran" : null,
+            data.getServiceEligibilityOtherRequirements(),
+            StringUtils.equalsIgnoreCase(data.getServiceEligibilitySocialSecurityCard(), TRUE) ? "Social Security Card Required" : null
+        ).filter(StringUtils::isNotBlank).collect(Collectors.joining(", "));
+        if (StringUtils.isBlank(eligibilityString)) {
+            return Optional.empty();
+        }
+        return Optional.of(new Eligibility().eligibility(eligibilityString));
+    }
+
+    default Organization extractOrganization(LinkForCareData data) {
+        Organization organization = new Organization();
+
+        if (StringUtils.isBlank(data.getOrganizationName())) {
+            LOG.warn("Name is empty for organization with ID: " + data.getOrganizationId());
+        } else {
+            organization.setName(data.getOrganizationName());
+        }
+        if (StringUtils.isNotBlank(data.getOrganizationCreatedDate())) {
+            organization.setYearIncorporated(ZonedDateTime.parse(data.getOrganizationCreatedDate()).toLocalDate());
+        }
+        if (StringUtils.isNotBlank(data.getOrganizationModifiedDate())) {
+            organization.setUpdatedAt(ZonedDateTime.parse(data.getOrganizationCreatedDate()));
+        }
+        organization.setUrl(data.getOrganizationWebsite());
+        organization.setDescription(data.getOrganizationDescription());
+        organization.setExternalDbId(data.getOrganizationId());
+        organization.setActive(true);
+        organization.setEmail(data.getOrganizationMainEmailAddress());
+        String covid19Protocols = Stream.of(data.getCovid19Protocols(), data.getCovid19ProtocolsFomula())
+            .filter(StringUtils::isNotBlank)
+            .collect(Collectors.joining(", "));
+        organization.setCovidProtocols(covid19Protocols);
+
+        return organization;
+    }
+
+    default Location extractLocation(LinkForCareData data) {
+        Location location = new Location();
+
+        location.name(String.join(", ", data.getLocationAddress1(),
+            data.getLocationCity(), data.getLocationState()));
+
+        location.setExternalDbId(data.getOrganizationId());
+        location.setProviderName(LINK_FOR_CARE_PROVIDER);
+        RegularSchedule regularSchedule = extractRegularSchedule(data);
+        location.setRegularSchedule(regularSchedule);
+        // TODO: set location remote service checkbox once #1321 is done
```

Comment:
Additionally remember, that all new fields (remote, fees, covidProtocols and 24/7) should be copied during the claiming (cloning) process here: https://github.com/benetech/ServiceNet/blob/c481399376e0ae868afb47f478ab602b7b6b0e7a/src/main/java/org/benetech/servicenet/service/impl/OrganizationServiceImpl.java#L569